int32 anchor_lat    # [1/10th microdegree]
int32 anchor_lon    # [1/10th microdegree]